src/main/java/frc/robot/RobotContainer.java

```diff
@@ -206,9 +208,8 @@ private void configureButtonBindings() {
     coDriverStick.btn_RBump.whenPressed(com_spinFlywheelGoalRPM);
     coDriverStick.btn_LBump.whileHeld(com_moveTurret);
 
-    // Limelight Commands
+    // Limelight Command
     coDriverStick.btn_A.whileHeld(com_visionAimTurret);
```

Comment:
`com_visionAimTurret` is already the default command of turret and is also the modifier button inside of `com_visionAimTurret`. So this line can/should be removed